controller/work_item_comments.go

```diff
@@ -74,32 +74,35 @@ func (c *WorkItemCommentsController) Create(ctx *app.CreateWorkItemCommentsConte
 
 		err = appl.Comments().Create(ctx, &newComment, *currentUserIdentityID)
 		if err != nil {
-			return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
+			return goa.ErrInternal(err.Error())
 		}
 
 		res := &app.CommentSingle{
 			Data: ConvertComment(ctx.Request, newComment),
 		}
 		return ctx.OK(res)
```

Comment:
I wonder if this `ctx.OK(res)` should maybe be move outside the transactional scope. But it appears as `nil` to the outside so it should be fine.